roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -12,13 +12,21 @@
       - ceph_docker_on_openstack
 
   - name: test if the container image has the disk_list function
-    command: "{{ container_binary }} run --rm --net=host --entrypoint=stat {{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }} disk_list.sh"
+    command: "{{ container_binary }} run --rm --net=host --entrypoint=stat {{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }} /opt/ceph-container/disk_list.sh"
     changed_when: false
     failed_when: false
     register: disk_list
     when:
       - osd_scenario != 'lvm'
 
+  - name: test if the container image has the old disk_list function (backward compatibility)
```

Comment:
could be nice to think to a nicer approach

Reply:
perhaps a loop?

Reply:
We could introduce a new variable for the container bin directory path (default to /opt/ceph-container/bin) and then run a test -d {dir} on the container image. If the directory isn't present we set the path to '' or '/'.
WDYT ?

Reply:
sounds like a good idea